tests/test_boards.py

```diff
@@ -1538,6 +1538,8 @@ def test_release_events_fire(mock_factory, capsys):
         bb.when_pressed = lambda: print("bb pressed", tuple(bb.value))
         bb.when_released = lambda: print("bb released", tuple(bb.value))
         pins[0].drive_low()
-        assert capsys.readouterr().out == "bb pressed (1, 0)\n"
+        out, _ = capsys.readouterr()
+        assert out == "bb pressed (1, 0)\n"
         pins[0].drive_high()
-        assert capsys.readouterr().out == "bb released (0, 0)\n"
+        out, _ = capsys.readouterr()
+        assert out == "bb released (0, 0)\n"
```

Comment:
I'm sure there has to be a nicer way of doing this than by printing a string, and trying to read back the string that got printed. Can't you just set a variable or something? :man_shrugging:

Reply:
Could do, I was just trying to keep it as close as possible to how the issue was described in #761